usb-devices

```diff
@@ -57,6 +57,11 @@ print_endpoint() {
         dir=B
     fi
 	eptype=`cat $eppath/type`
+    if [ "$eptype" = Control ]; then
+        eptype=Ctrl
```

Comment:
"Ctrl"?

Reply:
Exactly, see `usb_dump_endpoint_descriptor()` (checking in Linux 4.14.157 tree):
```c
	switch (usb_endpoint_type(desc)) {
	case USB_ENDPOINT_XFER_CONTROL:
		type = "Ctrl";
```

Reply:
Sorry for not being explicit, I was referring to the use of "" here, but not in the lines below for other assignments to this variable.  Seems like it should be either one way or the other, but not both, right?

Reply:
Oh indeed. The whole reason for using quotes for `Int.` was because it psychologically seems to not be just a single word, although `.` indeed isn't a word splitting character. I'll amend for consistency.

Reply:
Pushed now.